### PR TITLE
Milestone 4: Add styling

### PR DESCRIPTION
**Pull request by binyamolango**
# Hi Reviewer 👋

> Here are the changes that I implemented to the project

- Added any missing JSX elements to my project like the progress bar.
- Matched Zeplin's design which includes all elements + the same styling.
- Used the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/NavBar.js [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
-      </nav>
+    <header className={styles.header}>
+      <div className={styles.wrapper}>
+        <h1 className={styles.heading_text}>Bookstore CMS</h1>
+        <nav className={styles.navigation}>
+          <ul>
+            <li><Link className="nav-link" to="/">Books</Link></li>
+            <li><Link className="nav-link" to="/Categories">Categories</Link></li>
+          </ul>
+        </nav>
```
Comment (iambenkis) [inline src/components/NavBar.js:15]: - [ ] As for this project 👍 your design should be exaactly as presented in the [Zephin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) , Kindly make sure the Logo and the navlinks are well aligned 👍 

- [ ] Kindly uppercase the navlinks text 👍 to stick to the figma design

### Your design

Capture

### Zephin design

Capture

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @binyamolango 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️  Follow the Ziphin design 
✔️ Work documented professionally

## Suggested changes

- [ ] Good job documenting your project, it could be better to remove the `note`  text , it will make your README more professional 👍 

- [ ] Kindly fix the broken link attached to the issue page text , it will make your README more professional 😄 

Capture

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by binyamolango** [comment]
Hi @iambenkis, Thank you for reviewing my project and suggesting insightful changes. I managed to implement all of the required changes except for fixing the broken link attached to the issue page text part. The problem is that it can't be fixed. That link works correctly on the default branch which is the "dev" branch and doesn't work for the featured branches.

**Review by peterrobert — APPROVED** [review]
Hi @binyamolango, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️
[ ] Great work making the changes requested by the previous reviewer ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by binyamolango** [comment]
@peterrobert Thank you for reviewing and approving my project.

**Pull request merged**